app/models/tezos/block.rb

```diff
@@ -32,4 +32,22 @@ def timestamp
     update_columns(timestamp: time)
     return time
   end
+
+  def endorsers
+    return self[:endorsers] if self[:endorsers].present?
+
+    endorsers_from_rpc = []
+    url = Tezos::Rpc.new(chain).url("blocks/#{height}/helpers/endorsing_rights")
+    res = Typhoeus.get(url)
```

Comment:
Do we have a client for Tezos? Adding this low level Typhoeus call in the model might be problematic in the long run. Using client "service" would be better since we would need to make changes only in one place rather than searching through the code trying to find references.

Reply:
Yeah I can do a little refactoring. I already have another task for adding a test, so I would prefer to add that to the other ticket and get this merged first since it will prevent an issue we have seen in the live indexer.